interfaces/aux_/aux_.yaml

```diff
@@ -20,6 +20,59 @@ components:
           description: Human-readable message indicating what error occurred and/or why.
           type: string
 
+    DSSInstancesResponse:
+      type: object
+      properties:
+        dss_instances:
+          type: array
+          items:
+            $ref: '#/components/schemas/DSSInstance'
+          default: []
+    DSSInstance:
+      type: object
+      properties:
+        instance_id:
+          description: Identity of this DSS instance participating in the pool (locality).
+          type: string
+        most_recent_heartbeat:
+          description: Most recent heartbeat registered for this DSS instance.
+          anyOf:
+          - $ref: '#/components/schemas/Heartbeat'
+      required:
+      - instance_id
+    Heartbeat:
+      type: object
+      properties:
+        timestamp:
+          description: Time at which heartbeat was registered.
+          type: string
+          format: date-time
+          example: '1985-04-12T23:20:50.52Z'
+        reporter:
+          description: Identity (via access token `sub` claim) of client reporting the heartbeat, or omitted if no client reported the heartbeat.
+          type: string
+          example: 'uss1'
+          default: ''
+        source:
+          description: Source/trigger of this heartbeat.
+          type: string
+          example: Startup
+        index:
+          description: Index of this heartbeat within the set of all heartbeats for this pool participant.
+          type: integer
+          format: int64
+          minimum: 0
+          example: 1
+          default: 0
+        unique_value:
```

Comment:
My understanding of a heartbeat is that it is a signal of liveliness from the instance. With that understanding, I'm not sure this unique value should actually be part of the heartbeat?
In addition, how would that be used? IIUC comparing the last timestamps of heartbeats would be equivalent.

Reply:
My intent was to include the option of transmitting a "hard-to-guess value" as stronger evidence of synchronization than just transmitting a timestamp, but it seems like the timestamp would be sufficiently hard to guess as to make this field essentially redundant -- I'll remove.